edgegateway.go

```diff
@@ -307,5 +363,136 @@ func (e *EdgeGateway) Create1to1Mapping(internal, external, description string)
 
 	// The request was successful
 	return *task, nil
+}
+
+func (e *EdgeGateway) RequestPublicIP(networkname string, publicipcount string) (Task, error) {
+
+	// Refresh EdgeGateway rules
+	err := e.Refresh()
+	if err != nil {
+		fmt.Printf("error: %v\n", err)
+	}
+
+	gatewayInterfaces := e.EdgeGateway.Configuration.GatewayInterfaces.GatewayInterface
+
+	network := &types.Reference{}
+	for _, gatewayInterface := range gatewayInterfaces {
+		if gatewayInterface.Name == networkname {
+			network = gatewayInterface.Network
+			break
+		}
+	}
+
+	if network == nil {
+		log.Fatalf("err: couldn't find network name")
+	}
+
+	externalIPAddressActionList := &types2.ExternalIpAddressActionList{}
+	externalIPAddressActionList.Xmlns = "http://www.vmware.com/vcloud/networkservice/1.0"
+	externalIPAddressActionList.Allocation = &types2.Allocation{
+		ExternalNetworkName:                   network.Name,
+		ExternalNetworkRef:                    network.HREF,
+		NumberOfExternalIPAddressesToAllocate: publicipcount,
+	}
+
+	output, err := xml.MarshalIndent(externalIPAddressActionList, "  ", "    ")
+	if err != nil {
+		fmt.Printf("error: %v\n", err)
+	}
+
+	debug := os.Getenv("GOVCLOUDAIR_DEBUG")
```

Comment:
Make this consistent with the other envvar, make them both `GOVCLOUDAIR_DEBUG` or `VCLOUDAIR_DEBUG`